<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="dev_user" tests="4" time="0.187">
  <testsuite name="用户注册1.3" id="8a93aff9-1b96-4701-b1a7-8dacac8a246a" timestamp="2023-02-11T08:07:40.235Z" tests="2" failures="0" errors="0" time="0.138">
    <testcase name="Response time is less than 1000ms" time="0.138" classname="DevUser"/>
    <testcase name="Status code is 200" time="0.138" classname="DevUser"/>
  </testsuite>
  <testsuite name="获取验证码" id="5f4f1cd4-a95e-4c8d-af67-9e38b458e74f" timestamp="2023-02-11T08:07:40.235Z" tests="2" failures="0" errors="0" time="0.025">
    <testcase name="Response time is less than 1000ms" time="0.025" classname="DevUser"/>
    <testcase name="Status code is 200" time="0.025" classname="DevUser"/>
  </testsuite>
  <testsuite name="用户列表1.0" id="918d92c7-0210-4be1-b11b-d34363beb712" timestamp="2023-02-11T08:07:40.235Z" tests="2" failures="0" errors="0" time="0.012">
    <testcase name="Response time is less than 1000ms" time="0.012" classname="DevUser"/>
    <testcase name="Status code is 200" time="0.012" classname="DevUser"/>
  </testsuite>
  <testsuite name="用户登录1.3" id="05ba06d9-df8e-4063-b818-b936ff414371" timestamp="2023-02-11T08:07:40.235Z" tests="3" failures="1" errors="1" time="0.012">
    <system-err><![CDATA[Iteration: 0

---
testScriptError: AssertionFailure: Your test name
    at postProcessContext (C:\Users\dreamfuture\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:40:17)
    at PostmanCollectionRun.<anonymous> (C:\Users\dreamfuture\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:488:31)
    at PostmanSandbox.<anonymous> (C:\Users\dreamfuture\AppData\Roaming\npm\node_modules\newman\node_modules\postman-sandbox\lib\index.js:107:13)
    at Object.onceWrapper (node:events:628:26)
    at PostmanSandbox.emit (node:events:513:28)
    at context.__uvm_emit (C:\Users\dreamfuture\AppData\Roaming\npm\node_modules\newman\node_modules\uvm\lib\bridge.js:71:21)
    at Object.dispatch (evalmachine.<anonymous>:20:13)
    at Object.<anonymous> (evalmachine.<anonymous>:58:7101)
    at t.exports.terminate (evalmachine.<anonymous>:58:18217)
    at evalmachine.<anonymous>:58:5853
    at evalmachine.<anonymous>:58:1789089
    at u.exec (evalmachine.<anonymous>:58:1788668)
    at t.exports (evalmachine.<anonymous>:58:5743)
    at Object.<anonymous> (evalmachine.<anonymous>:58:7440)
    at evalmachine.<anonymous>:15:26
    at Array.forEach (<anonymous>)
---
]]></system-err>
    <testcase name="Response time is less than 1000ms" time="0.012" classname="DevUser"/>
    <testcase name="Status code is 200" time="0.012" classname="DevUser"/>
    <testcase name="Your test name" time="0.012" classname="DevUser">
      <failure type="AssertionFailure" message="expected '用户账户或密码错误，登录失败' to deeply equal '成功'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: a08a8fa7-c02b-44d2-b4ac-72d13c0739d0.]]>
        <![CDATA[Collection name: dev_user.]]>
        <![CDATA[Request name: 用户登录1.3.]]>
        <![CDATA[Test description: Your test name.]]>
        <![CDATA[Error message: expected '用户账户或密码错误，登录失败' to deeply equal '成功'.]]>
        <![CDATA[Stacktrace: AssertionError: expected '用户账户或密码错误，登录失败' to deeply equal '成功'
   at Object.eval sandbox-script.js:3:4).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>